linux.go

```diff
@@ -22,13 +23,26 @@ func NewLinuxTracker() Tracker {
 	return &LinuxTracker{}
 }
 
-func (t *LinuxTracker) Deps() string {
-	return `Install the following command-line utilities via your package manager (e.g., apt) of choice:
-* xdpyinfo
-* xwininfo
-* xdotool
-* wmctrl
-`
+func (t *LinuxTracker) CheckDependencies() {
+	deps := map[string]string{
+		"xdpyinfo": "x11-utils",
+		"xwininfo": "x11-utils",
```

Comment:
Note that package names are different for RPM, as in https://github.com/sourcegraph/thyme/pull/48